docs/spec/spec.md

```diff
@@ -197,52 +225,158 @@ channel._
 absolute URL. If both are specified, uri will be resolved using the base URI
 retrieved from ref.
 
-### duck.DeliverySpec
+### eventingduckv1.DeliverySpec
 
-| Field          | Type          | Description                                                                                                                                      | Constraints |
-| -------------- | ------------- | ------------------------------------------------------------------------------------------------------------------------------------------------ | ----------- |
-| deadLetterSink | Destination   | DeadLetterSink is the sink receiving event that could not be sent to a destination.                                                              |             |
-| retry          | String        | Retry is the minimum number of retries the sender should attempt when sending an event before moving it to the dead letter sink.                 |             |
-| backoffPolicy  | BackoffPolicy | BackoffPolicy is the retry backoff policy (linear, exponential).                                                                                 |             |
-| backoffDelay   | String        | For linear policy, backoff delay is backoffDelay\*<numberOfRetries>. For exponential policy, backoff delay is backoffDelay\*2^<numberOfRetries>. |             |
+`DeliverySpec` contains the delivery options for event senders.
 
-### duck.SubscriberSpec
+| Field Name       | Field Type                                  | Requirement | Description                                                                                                                                                       | Constraints |
+| ---------------- | ------------------------------------------- | ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------- |
+| `deadLetterSink` | [`duckv1.Destination`](#duckv1.Destination) | Optional    | The sink receiving event that could not be sent to a `Destination`.                                                                                               |             |
+| `retry`          | `int`                                       | Optional    | The minimum number of retries the sender should attempt when sending an event before moving it to the dead letter sink (when specified) or discarded (otherwise). |             |
+| `backoffPolicy`  | `string`                                    | Optional    | The retry backoff policy (`linear` or `exponential`).                                                                                                             |             |
+| `backoffDelay`   | `string`                                    | Optional    | For linear policy, backoff delay is backoffDelay\*<numberOfRetries>. For exponential policy, backoff delay is backoffDelay\*2^<numberOfRetries>.                  |             |
```

Comment:
This will have the same problem as in PR #4690. Can you please make sure this PR will be applied here too?